docs/docs/what-about-generators.md

```diff
@@ -3,4 +3,5 @@ id: what-about-generators
 title: What About Generators?
 ---
 
```

Comment:
Use `\[Promise.coroutine\]\(.\)` and it will automatically build up API link from it